modules/viz/misc/python/test/test_simple.py

```diff
@@ -0,0 +1,436 @@
+#exec(open(r'G:\Lib\opencv_contrib\modules\viz\samples\viz_sample_03.py').read())
+import cv2 as cv
+import numpy as np
+
+from tests_common import NewOpenCVTests
+
+def generate_test_trajectory():
+    result =  []
+    angle_i = np.arange(0, 271, 3)
+    angle_j = np.arange(0, 1200, 10)
+    for i, j in zip(angle_i, angle_j):
+        x = 2 * np.cos(i * 3 * np.pi/180.0) * (1.0 + 0.5 * np.cos(1.2 + i * 1.2 * np.pi/180.0))
+        y = 0.25 + i/270.0 + np.sin(j * np.pi/180.0) * 0.2 * np.sin(0.6 + j * 1.5 * np.pi/180.0)
+        z = 2 * np.sin(i * 3 * np.pi/180.0) * (1.0 + 0.5 * np.cos(1.2 + i * np.pi/180.0))
+        result.append(cv.viz.makeCameraPosePy((x, y, z), (0.0, 0, 0), (0.0, 1.0, 0.0)))
+    x =  np.zeros(shape=(len(result), 1, 16 ), dtype= np.float64)
+    for idx, m in enumerate(result):
+        x[idx, 0, :] = m.mat().reshape(16)
+    return x, result
+
+def tutorial3(camera_pov, filename):
+    myWindow = cv.viz_Viz3d("Coordinate Frame")
+    myWindow.showWidget("axe",cv.viz_PyWCoordinateSystem())
+
+    cam_origin =  (3.0, 3.0, 3.0)
+    cam_focal_point = (3.0,3.0,2.0)
+    cam_y_dir = (-1.0,0.0,0.0)
+    camera_pose = cv.viz.makeCameraPosePy(cam_origin, cam_focal_point, cam_y_dir)
+    transform = cv.viz.makeTransformToGlobalPy((0.0,-1.0,0.0), (-1.0,0.0,0.0), (0.0,0.0,-1.0), cam_origin)
+    dragon_cloud,_,_ = cv.viz.readCloud(filename)
+    cloud_widget = cv.viz_PyWCloud(dragon_cloud, cv.viz_PyColor().green())
+    cloud_pose = cv.viz_PyAffine3()
+    cloud_pose = cv.viz_PyAffine3().rotate((0, np.pi / 2, 0)).translate((0, 0, 3))
+    cloud_pose_global = transform.product(cloud_pose)
+    myWindow.showWidget("CPW_FRUSTUM", cv.viz_PyWCameraPosition((0.889484, 0.523599)), camera_pose)
+    if not camera_pov:
+        myWindow.showWidget("CPW", cv.viz_PyWCameraPosition(0.5), camera_pose)
+    myWindow.showWidget("dragon", cloud_widget, cloud_pose_global)
+    if camera_pov:
+        myWindow.setViewerPosePy(camera_pose)
+
+class viz_test(NewOpenCVTests):
+
```

Comment:
Lets skip VIZ tests by default (because they are UI tests):

```
    def setUp(self):
        super(viz_test, self).setUp()
        if not bool(os.environ.get('OPENCV_PYTEST_RUN_VIZ', False)):
            self.skipTest("Use OPENCV_PYTEST_RUN_VIZ=1 to enable VIZ UI tests")
```